src/tamil/README.md

```diff
@@ -0,0 +1,25 @@
+# Tamil
+
+## Language info
+
+- English name: Tamil
+- Autonym: Tam
```

Comment:
Autonym should be the name of the language in the language itself, so தமிழ் (according to Wikipedia)